packages/primitives/src/icons/diamond-16.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's add the license header

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/220/commits/5308d2970b6e2e16934a4084597c982455ed8c6d